terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -13,13 +13,273 @@
+    (OTHER, _("other")),
+)
+
+GRANULAR = "granular"
+SUBANGULAR_BLOCKY = "subangular blocky"
+ANGULAR_BLOCKY = "angular blocky"
+LENTICULAR = "lenticular"
+PLAY = "play"
+WEDGE = "wedge"
+PRISMATIC = "prismatic"
+COLUMNAR = "columnar"
+SINGLE_GRAIN = "single grain"
+MASSIVE = "massive"
+
+STRUCTURES = (
+    (GRANULAR, _("Granular")),
+    (SUBANGULAR_BLOCKY, _("Subangular Blocky")),
+    (ANGULAR_BLOCKY, _("Angular Blocky")),
+    (LENTICULAR, _("lenticular")),
+    (PLAY, _("Play")),
+    (WEDGE, _("Wedge")),
+    (PRISMATIC, _("Prismatic")),
+    (COLUMNAR, _("Columnar")),
+    (SINGLE_GRAIN, _("Single Grain")),
+    (MASSIVE, _("Massive")),
+)
+
+PH_TESTING_SOIL_WATER_1_1 = "1:1 soil/water"
+PH_TESTING_SOIL_WATER_1_2 = "1:2 soil/water"
+PH_TESTING_SOIL_WATER_1_2_5 = "1:2.5 soil/water"
+PH_TESTING_SOIL_WATER_1_5 = "1:5 soil/water"
+PH_TESTING_SOIL_CACL2_1_1 = "1:1 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_CACL2_1_2 = "1:2 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_CACL2_1_5 = "1:5 soil/0.1 M CaCL2"
+PH_TESTING_SOIL_KCL_1_1 = "1:1 soil/1.0 M KCL"
+PH_TESTING_SOIL_KCL_1_2_5 = "1:2.5 soil/1.0 M KCL"
+PH_TESTING_SOIL_KCL_1_5 = "1:5 soil/1.0 M KCL"
+PH_TESTING_SATURATED_PASTE_EXTRACT = "saturated paste extract"
+
+PH_TESTING_SOLUTIONS = (
+    (PH_TESTING_SOIL_WATER_1_1, _("1:1 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_2, _("1:2 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_2_5, _("1:2.5 soil/water")),
+    (PH_TESTING_SOIL_WATER_1_5, _("1:5 soil/water")),
+    (PH_TESTING_SOIL_CACL2_1_1, _("1:1 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_CACL2_1_2, _("1:2 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_CACL2_1_5, _("1:5 soil/0.1 M CaCL2")),
+    (PH_TESTING_SOIL_KCL_1_1, _("1:1 soil/1.0 M KCL")),
+    (PH_TESTING_SOIL_KCL_1_2_5, _("1:2.5 soil/1.0 M KCL")),
+    (PH_TESTING_SOIL_KCL_1_5, _("1:5 soil/1.0 M KCL")),
+    (PH_TESTING_SATURATED_PASTE_EXTRACT, _("Saturated Paste Extract")),
+    (OTHER, _("Other")),
+)
+
+
+PH_TESTING_INDICATOR_STRIP = "pH indicator strip"
+PH_TESTING_INDICATOR_SOLUTION = "pH indicator solution"
+PH_TESTING_METER = "pH meter"
+
+PH_TESTING_METHODS = (
+    (PH_TESTING_INDICATOR_STRIP, _("pH indicator strip")),
+    (PH_TESTING_INDICATOR_SOLUTION, _("pH indicator solution")),
+    (PH_TESTING_METER, _("pH meter")),
+    (OTHER, _("other")),
+)
+
+SOIL_TESTING_DRY_COMBUSTION = "Dry combustion"
+SOIL_TESTING_WET_OXIDATION = "Wet oxidation (Walkey-Black)"
+SOIL_TESTING_LOSS_ON_IGNITION = "Loss-on-ignition"
+SOIL_TESTING_REFLECTANCE_SPECTROSCOPY = "Reflectance spectroscopy"
+SOIL_TESTING_FIELD_REFLECTOMETER = "Field reflectometer"
+
+SOIL_TESTING_METHODS = (
+    (SOIL_TESTING_DRY_COMBUSTION, _("Dry combustion")),
+    (SOIL_TESTING_WET_OXIDATION, _("Wet oxidation (Walkey-Black)")),
+    (SOIL_TESTING_LOSS_ON_IGNITION, _("Loss-on-ignition")),
+    (SOIL_TESTING_REFLECTANCE_SPECTROSCOPY, _("Reflectance spectroscopy")),
+    (SOIL_TESTING_FIELD_REFLECTOMETER, _("Field reflectometer")),
+    (OTHER, _("Other")),
+)
+
```

Comment:
nit: we usually put the choices in the actual body of the model they're for, and place them right before the relevant field, to keep things tidier and so all the constants don't all have to be imported individually

Reply:
Sounds good! Easier to read too!